apps/detect/src/components/decision/decision-list.tsx

```diff
@@ -28,16 +31,42 @@ export default function DecisionList() {
     );
   }
 
+  data.forEach((host) => {
+    const {
+      error: errorCount,
+      protected: protectedCount,
+      unprotected: unprotectedCount,
+    } = host.volume;
+    if (unprotectedCount > 0 || errorCount > 0) {
+      unsecuredInsights.push(host);
+    }
+    if (protectedCount > 0 && unprotectedCount === 0) {
+      securedInsights.push(host);
+    }
```

Comment:
```suggestion
  const unsecuredInsights: = data.filter(.....)
  const securedInsights: Insight[] = data.filter(.....)
```
Its uses more memory since we are copying but its easier to chain and read IMO